src/js/module/toolbox/sidebar.js

```diff
@@ -1,58 +1,97 @@
-import underscore from 'underscore';
+import { chain, range, map } from 'underscore';
```

Comment:
I know sure for now, but it looks like: https://github.com/eslint/eslint/pull/4820